docs/sources/installation/docker.md

```diff
@@ -103,6 +105,8 @@ docker run \
   grafana:latest-with-plugins
 ```
 
+> If you need to specify the version of a plugin, you can add it to the build argument. Otherwise latest will be assumed. `--build-arg "GF_INSTALL_PLUGINS=grafana-clock-panel 1.0.1,grafana-simple-json-datasource 1.3.5"`
```

Comment:
Same here, I would change it to:
> If you need to specify the version of a plugin, you can add it to the `GF_INSTALL_PLUGINS` build argument. Otherwise, the latest will be assumed.
For example: `--build-arg "GF_INSTALL_PLUGINS=grafana-clock-panel 1.0.1,grafana-simple-json-datasource 1.3.5"`